content/docs/concepts/metric_types.md

```diff
@@ -48,6 +48,10 @@ A _histogram_ samples observations (usually things like request durations or
 response sizes) and counts them in configurable buckets. It also provides a sum
 of all observed values.
 
+Since each observation bucket holds the total count of observations less than or
```

Comment:
A couple of lines further down, it is mentioned that the counters are cumulative. "Cumulative histogram" is a real thing: https://en.wikipedia.org/wiki/Histogram#Cumulative_histogram
Perhaps it would be sufficient to stress the fact that the histograms in Prometheus are cumulative a bit more prominently and further up.

The implications of that are manifold, blacklisting is only one of them, so I would not go into that kind of detail here.